ServiceWebsite/ServiceWebsite.AcceptanceTests/Configuration/ServiceWebConfig.cs

```diff
@@ -1,13 +1,15 @@
 ﻿using AcceptanceTests.Common.Configuration;
+using ServiceWebsite.Configuration;
 
 namespace ServiceWebsite.AcceptanceTests.Configuration
 {
     public class ServiceWebConfig
     {
-        public ServiceWebSecurityConfiguration AzureAdConfiguration { get; set; }
+        public SecuritySettings AzureAdConfiguration { get; set; }
         public bool IsLive { get; set; }
         public ServiceWebTestConfig TestConfig { get; set; }
         public ServiceWebVhServicesConfig VhServices { get; set; }
         public SauceLabsSettingsConfig SauceLabsConfiguration { get; set; }
+        public bool UsingEjud { get; set; }
```

Comment:
Are judges expected to log intoService web with their EJud accounts now? I didn't expect so, if so please remove it.